glide.yaml

```diff
@@ -4,6 +4,7 @@ import:
   verison: 496d495156da218b9912f03dfa7df7f80fbd8cc3
 - package: github.com/fabric8io/golang-jenkins
```

Comment:
If this is a forked repo (which I think it is) you're better off using the proper package name (`github.com/yosida95/golang-jenkins`) & setting the repo path, something like this:

```
- package: github.com/yosida95/golang-jenkins
  vcs: git
  repo: git@github.com:fabric8io/golang-jenkins.git
  version: BRANCH/COMMIT/TAG/WHATEVER 
```

This is all handled by glide about v0.10+ I think.